README.md

```diff
@@ -30,62 +30,58 @@ This tool provides the following features:
 
 ### Configuring the Tool
 
-We use a YAML file to configure this Tool in order to create the Spark and LLAP HDI Cluster's as part of Hive Warehouse Connector in the same VNet.
-
- - The below YAML config file should be passed to this tool inorder to initiate the HWC cluster creation.
- - The below configs shows all the configuration available for running this tool. You can also refer the conf folder inside this repo which has templates for standard and secure cluster configs.
+User can set configs in a YAML file (as shown below) and pass it to the tool. The below list covers all the configurations supported by this tool. Additionally, the conf folder in this repo has templates for standard and secure cluster config files
  
 ```yml
-type: SPARK_AND_LLAP # Optional, values can be either SPARK_ONLY or LLAP_ONLY. Default is SPARK_AND_LLAP
-clusterNamePrefix: Foo # This prefix is used while creating the cluster name. Only first three chars are used as prefix from this string.
-resourceGroup: foobar-rg # Resource group where the cluster needs to be created
-region: eastus2 # Region name where the cluster needs to be created, should be in smallcase without space 
-headNodeVMSize: STANDARD_D13_V2 # Any Standrad VM Size supported for Head Nodes
-workerNodeVMSize: STANDARD_D13_V2 # Any Standrad VM Size supported for Worker Nodes
+type: SPARK_AND_LLAP # [Optional], values can be either SPARK_ONLY or LLAP_ONLY Default is SPARK_AND_LLAP
```

Comment:
How user can input the clustername of already existing cluster?

Reply:
If there is a Spark or LLAP cluster already existing, then user just needs to mention `LLAP_ONLY` or `SPARK_ONLY` in the type and give the VNet information of the existing cluster.

_For example_, if the user has an existing **LLAP cluster**, they can use this Tool to create **Spark Cluster** by specifying the type as `SPARK_ONLY` and configure the network with the existing VNet of the **LLAP Cluster** by setting **`create`** field to false.

**Example Config**
```
type: SPARK_ONLY
...
...
...
network:
  vnetName: testvnet
  subnetName: testsubnet
  resourceGroup: test-rg
  create: false
```